.ko.yaml

```diff
@@ -1 +1,31 @@
-defaultBaseImage: gcr.io/k8s-prow/git:v20211018-372b1c8
+# Distroless images:
+# - prow/cmd/admission
+# - prow/cmd/gerrit
+# - prow/cmd/mkpj
+# - prow/cmd/mkpod
+# - prow/cmd/pipeline
```

Comment:
Will these still be capable of being execed into so that we can debug with a shell? I suspect it was an oversight that these are not using the alpine base.

Reply:
These images use distroless base, so they cannot be execed into. I'd prefer stick with what they are within current PR, as this is what's been used in prod today. I can change them to alpine later once confirmed that everything works